### DISCUSSION TITLE
Does “GitHub mode” mean it’s framework agnostic?

### DISCUSSION THREAD

**Discussion by m1chaeljmk** [Thinkmill/keystatic#686] [Q&A] [answered]
Sorry for the beginner level question, but does connecting with GitHub mean I could use keystatic to edit an 11ty site, for example?

**Comment by simonswiss** [comment, marked as answer]
Hey!

Yes, using the `github` mode lets you use the framework of your choice.

The `github` mode means that Keystatic will be connected to a GitHub repo, and when you make changes to content, those changes will generate commits on a given branch.

This is totally decoupled from what framework you use for your front end.

A pretty common setup is to use `local` mode when running a Keystatic project locally, and set the deployed site to use the `github` mode — that's what I do on most my projects 👍

Hope this helps!
